generators/client/templates/src/main/webapp/app/admin/user-management/_user-management.controller.js

```diff
@@ -71,8 +50,45 @@
                 lastModifiedBy: null, lastModifiedDate: null, resetDate: null,
                 resetKey: null, authorities: null
             };
-            vm.editForm.$setPristine();
```

Comment:
I didn't see a vm.editForm variable and activating/deactivating a user threw an error in the console saying vm.editForm is undefined, so I removed these.